database/migrations/2025_06_10_220804_user_invitation_ttl.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * SPDX-License-Identifier: MIT
+ * Copyright (c) 2017-2018 Tobias Reich
+ * Copyright (c) 2018-2025 LycheeOrg.
+ */
+
+use App\Models\Extensions\BaseConfigMigration;
+
+return new class() extends BaseConfigMigration {
+	public const CAT = 'Users Management';
+
+	public function getConfigs(): array
+	{
+		return [
+			[
+				'key' => 'user_invitation_ttl',
+				'value' => '7',
+				'cat' => self::CAT,
+				'type_range' => self::BOOL,
+				'description' => 'Maximum life time for invitation links in days.',
```

Comment:
<img width="882" alt="image" src="https://github.com/user-attachments/assets/7f5a12e3-bba6-47ec-8bb9-709733c44177" />


This is confusing for me as a user, how do I set the life time or if I don't it would be a good idea to explicitly state the number of days.

Reply:
Fixed [9539700](https://github.com/LycheeOrg/Lychee/pull/3433/commits/95397008c9b308e5584f4a797fbd5667a532a71e)